async-nats/src/jetstream/consumer/mod.rs

```diff
@@ -147,14 +147,14 @@ pub struct Info {
     pub delivered: SequenceInfo,
     /// Statistics for acknowledged messages
     pub ack_floor: SequenceInfo,
-    /// The difference between delivered and acknowledged messages
+    /// The number of messages delivered but not yet acknowledged
     pub num_ack_pending: usize,
     /// The number of messages re-sent after acknowledgment was not received within the configured
     /// time threshold
     pub num_redelivered: usize,
-    /// The number of waiting
+    /// The number of clients waiting for messages from the consumer
```

Comment:
That is not exactly accurate - one client can send more than one Pull Request to the server.

I would probably just say that this is the number of pending pull requests sent by clients.

Reply:
Changed 👍